src/components/ImageView/index.native.js

```diff
@@ -120,8 +135,9 @@ class ImageView extends PureComponent {
                     <Image
                         source={{uri: this.props.url}}
                         style={StyleUtils.getWidthAndHeightStyle(this.state.thumbnailWidth, this.state.thumbnailHeight)}
-                        resizeMode={Image.resizeMode.contain}
+                        resizeMode="contain"
```

Comment:
Was this change necessary?

Reply:
It was not necessary. I did this change to be consistent.